tests/fixtures/dag_factory.yml

```diff
@@ -1,11 +1,15 @@
 default:
   concurrency: 1
-  dagrun_timeout_sec: 600
+  dagrun_timeout:
+    __type__: datetime.timedelta
```

Comment:
Maybe we could make dag authoring simpler by accepting int values as the default getting converted to datetime.timedelta seconds, so that users won't have to specify a __type__ construct unless they want something different than the default. WDYT?